charts/kubearchive/values.yaml

```diff
@@ -91,3 +87,14 @@ database:
     dbUser: "ps_user"
     # database password
     dbPassword: "P0stgr3sdbP@ssword"  # notsecret
+
+observability:
+  # If "true" a Jaeger all-in-one Deployment is created in the '{{ .Release.Namespace }}' namespace
+  enabled: false
+  # Jaeger all-in-one image tag used for the Deploy if 'enabled' is true
+  imageTag: 1.57.0
+  # Value to which OTEL_EXPORTER_OTLP_ENDPOINT is set. If set KubeArchive tries to send telemetry data
+  # If you set `enabled` to true, use
```

Comment:
Why can't you just do this conditionally if that value is set?  I'd rather just set enabled to `true` and have it work, rather than having to read the rest of the comments to see that I have to do additional stuff.

Reply:
I changed it so if you set enabled: true whatever is on the endpoint is replaced by the endpoint of the jaeger Deployment. This way it will be easier to use in development. If enabled is false the value of endpoint is used to populate the variable.